modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+
+.Query
+[source, cypher]
+----
+DROP ALIAS `northwind` IF EXISTS FOR DATABASE
+----
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2020`
+----
+
+[[database-management-create-remote-database-alias]]
+=== Create remote database aliases
+
+A database alias can target a remote database by providing an URL and the credentials of a user on the remote Neo4j DBMS.
+See link:{neo4j-docs-base-uri}/operations-manual/{page-version}/manage-databases/remote-alias[Connecting remote databases] for the necessary configurations.
+
+ remote database aliases also allows `IF NOT EXISTS` and `OR REPLACE` clauses.
+Both check for any remote or local database aliases.
+
+
+==== Create remote database aliases
+
+//Skip testing all remote examples because it requires a lot of configuration, both server and client side.
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-northwind` FOR DATABASE `northwind-graph-2020`
+AT "neo4j+s://location:7687"
+USER alice
+PASSWORD 'example_secret'
+----
+
+When you create a database alias pointing to a remote database, its details can be shown with the `SHOW ALIASES FOR DATABASE` command.
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `remote-northwind`
+FOR DATABASE
+----
+
+.Result
+[role="queryresult"]
+----
++----------------------------------------------------------------------------------------------------------+
+| name               | composite | database               | location | url                       | user    |
++----------------------------------------------------------------------------------------------------------+
+| "remote-northwind" | NULL      | "northwind-graph-2020" | "remote" | "neo4j+s://location:7687" | "alice" |
++----------------------------------------------------------------------------------------------------------+
+----
+
+
+==== Create remote database aliases with driver settings
+
+It is possible to override the default driver settings per database alias, which are used for connecting to the remote database.
+
+This is the list of the allowed driver settings for remote database aliases:
+
+.dbms.routing.driver.ssl_enforced
+[width="100%", cols="1s, 4a"]
+|===
+| Description
+|
+SSL for remote database alias drivers is configured through the target URL scheme.
+If `ssl_enforced` is set to true, a secure URL scheme is enforced.
+This will be validated when the command is executed.
+
+| Valid values
+| Boolean
+
+| Default value
+| true
+
+|===
+
+.dbms.routing.driver.connection.connect_timeout
+[width="100%", cols="1s, 4a"]
+|===
+
+| Description
+|
+Socket connection timeout.
+A timeout of zero is treated as an infinite timeout and will be bound by the timeout configured on the operating system level.
+
+| Valid values
+| Duration
+
+| Default value
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings#config_dbms.routing.driver.connection.connect_timeout[dbms.routing.driver.connection.connect_timeout]
```

Comment:
once more a link hat maybe could get simplified as we are now in the operations manual

Reply:
there are more in the other settings below as well